ajax/libs/preact-compat/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "preact-compat",
+  "version": "3.15.0",
+  "filename": "preact-compat.min.js",
+  "description": "A React compatibility layer for Preact",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/developit/preact-compat.git"
+  },
+  "keywords": [
+    "preact",
+    "react",
+    "compatibility"
+  ],
+  "homepage": "https://github.com/developit/preact-compat",
```

Comment:
We'll use git repo as homepage by default if the custom homepage is not found, so there is no need to set homepage to its git repo.